src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -3824,12 +3878,12 @@ public boolean save() {
     return false;
   }
 
-  public void setConfigName(String configName) {
+  public String setConfigName(String configName) {
     if (configName == null || configName.trim().length() == 0) {

```

Comment:
StringUtils.isEmpty is probably a bit cleaner than this if test here.

Reply:
we don't pack StringUtils as part of base jar